src/modules/ui/overlay/overlay.ts

```diff
@@ -94,7 +110,14 @@ class Overlay implements IOverlay {
     }
   }
 
+  private _showControlsAndDescription(){
```

Comment:
better `showMainUIBlock`. but if we gonna remove `this._isControlsAndDescriptionHidden`, then there are no reasons for this method at all